pkgs/by-name/ve/versatiles/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "versatiles";
+  version = "0.12.9"; # When updating: Replace with current version
+
+  src = fetchFromGitHub {
+    owner = "versatiles-org";
+    repo = "versatiles-rs";
+    rev = "a1896bc1e639ec46bd00f89bffaba9181614515a"; # When updating: Replace with long commit hash of new version
```

```suggestion
    rev = "v${version}";
```

I think it's more or less convention to just use the tag here. The hash ensures that noone tampers with the tag.

---

Better
```suggestion
    rev = "refs/tags/v${version}";
```